--- openwrt-imagebuilder-21.02-SNAPSHOT-ath79-generic.Linux-x86_64/scripts/json_overview_image_info.py	2021-05-06 23:12:15.968882605 +0200
+++ openwrt-imagebuilder-21.02-SNAPSHOT-ath79-generic.Linux-x86_64/scripts/json_overview_image_info.py	2021-05-06 23:12:32.488834314 +0200
@@ -18,6 +18,9 @@
 
 output = {}
 
+if output_path.is_file():
+    output = json.loads(output_path.read_text())
+
 for json_file in work_dir.glob("*.json"):
     image_info = json.loads(json_file.read_text())
     if not output:
@@ -38,7 +41,7 @@
             "make",
             "--no-print-directory",
             "-C",
-            "target/linux/",
+            "target/linux/{}".format(output['target'].split('/')[0]),
             "val.DEFAULT_PACKAGES",
             "val.ARCH_PACKAGES",
         ],
@@ -49,8 +52,7 @@
         universal_newlines=True,
     ).stdout.splitlines()
 
-    output["default_packages"] = sorted(default_packages.split())
-
+    output["default_packages"] = default_packages.split()
     output_path.write_text(json.dumps(output, sort_keys=True, separators=(",", ":")))
 else:
     print("JSON info file script could not find any JSON files for target")
